### ISSUE TITLE
Add online help

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-emails#218]
Some users have hard time configuring a Gmail account (cf https://forum.cozy.io/t/impossible-dajouter-un-compte-gmail/357/6 ) We should find a way to display more help / a FAQ.

**Comment by aenario** [comment]
The Gmail doc at https://support.google.com/accounts/answer/6009563 is pretty clear & complete and automatically translated depending on Gmail's user language. I believe the "not compatible with Gmail security norm" is sufficiently objective to not scare users from cozy-emails.

I dont think we can do better, so we might as well link to it. (The imap error we receive from Gmail actually includes a short link to it).

**Comment by clochix** [comment]
Yes, for this specific case, a link to Gmail's doc would do the trick. But I think this isn't the only point where users may need some help, so imho we should think of a way to display inline help.
